src/components/Toggle/Toggle.module.css

```diff
@@ -0,0 +1,85 @@
+@import '../../design-tokens/mixins.css';
+
+.toggle__button {
+  border: 0;
+  padding: 0;
+
+  position: relative;
+  display: inline-flex;
+
+  height: var(--eds-size-3);
+  width: var(--eds-size-6);
+
+  flex-shrink: 0;
+  align-items: center;
+
+  border-radius: var(--eds-border-radius-full);
+
+  cursor: pointer;
+
+  transition: background-color var(--eds-anim-move-quick) var(--eds-anim-ease);
+
+  @media screen and (prefers-reduced-motion) {
+    transition: none;
+  }
+}
+
+.toggle__button:focus {
+  @mixin focus;
+}
+
+.toggle__button--unchecked {
+  background-color: var(--eds-theme-color-icon-neutral-subtle);
+}
+
+.toggle__button--checked {
+  background-color: var(--eds-theme-color-icon-brand-primary);
+}
+
+.toggle__button:disabled {
+  background-color: var(--eds-theme-color-icon-disabled);
+  cursor: not-allowed;
+}
+
+.toggle__button:disabled ~ .toggle__label {
+  color: var(--eds-theme-color-text-disabled);
+}
+
+.toggle__label {
+  margin-bottom: 0;
+
+  color: var(--eds-theme-color-text-neutral-default);
+  @mixin eds-theme-typography-body-text-md;
+}
+
+.toggle__wrapper {
+  display: flex;
+  align-items: center;
+
+  gap: var(--eds-size-1);
+}
+
+.toggle__button-circle {
```

Comment:
I'm also wondering if we should have a transparent border on the circle for the same reason mentioned on the button.

Reply:
I can add since visually wouldn't make a difference